packages/application-generic/src/logging/index.ts

```diff
@@ -131,6 +131,7 @@ export function createNestLoggingModuleOptions(
           organizationId: req?.user?.organizationId || null,
         },
         authScheme: req?.authScheme,
+        rateLimitPolicy: res?.rateLimitPolicy,
```

Comment:
Adding the rate limit policy to the logs for use in observability, specifically Rate Limit utilization.